app/components/Button.tsx

```diff
@@ -226,7 +226,6 @@ const $shadowPresets = {
     $baseShadowStyle,
     {
       backgroundColor: colors.palette.primary500,
-      flexGrow: 1,
```

Comment:
@jonmajorc LMK if this is an issue but it didn't seem to impact LinkButton, however it cleaned up the graphic displayed in the PR notes